orion-viewer/src/main/res/values-tr/langs.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <!--Do not translate this section and do not include it in result localization: start-->
+        <string name="lang_default" translatable="false">DEFAULT</string>
+        <string name="lang_english" translatable="false">en</string>
+        <string name="lang_russian" translatable="false">ru</string>
+        <string name="lang_hebrew" translatable="false">iw</string>
+        <string name="lang_chinese" translatable="false">zh</string>
+        <string name="lang_italian" translatable="false">it</string>
+        <string name="lang_spanish" translatable="false">es</string>
+        <string name="lang_ukrainian" translatable="false">ua</string>
+        <string name="lang_german" translatable="false">de</string>
+        <string name="lang_french" translatable="false">fr</string>
+        <string name="lang_turkish" translatable="false">tr</string>
```

Comment:
@symbuzzer Please remove this commit, just add 
`<string name="lang_turkish" translatable="false">tr</string>`
and 
`    <string name="lang_turkish_user">Turkish</string>`
to `orion-viewer/src/main/res/values/langs.xml`

Reply:
Please also update `orion-viewer/src/main/res/values/arrays.xml` file - `pref_languages` and `pref_languages_desc` arrays - with new translation

Reply:
> @symbuzzer Please remove this commit, just add `<string name="lang_turkish" translatable="false">tr</string>` and ` <string name="lang_turkish_user">Turkish</string>` to `orion-viewer/src/main/res/values/langs.xml`

It is already done?
![image](https://github.com/max-kammerer/orion-viewer/assets/9841184/72a6e802-9865-46e7-bd6b-83206efc696b)

Reply:
> Please also update `orion-viewer/src/main/res/values/arrays.xml` file - `pref_languages` and `pref_languages_desc` arrays - with new translation

Done

Reply:
>It is already done?

@symbuzzer No, I see that both commits  updating 'values-tr/langs.xml' are still present in pull request, they are redundant. Please just add
`<string name="lang_turkish" translatable="false">tr</string>`
and
` <string name="lang_turkish_user">Turkish</string>`
to default resource `.../values/langs.xml`

Reply:
Sorry for misunderstand. Done